src/main/resources/features/theme 1/17 - scheme user.feature

```diff
@@ -25,26 +25,56 @@ Feature: 17 - Scheme user
     And I enter {month} in the field with id "external-month"
     And I enter {year} in the field with id "external-year"
     And I enter "10" in the "Number of days to acknowledge by" field
-    And I enter "Test Special Notice" in the "Message title" field
+    And I enter "Test SN 1" in the "Message title" field
     And I enter "This is a test special notice." in the field with id "noticeText"
+
+    #Continue through journey and save as a draft and verify the page contains the correct values on the review page
     And I press the "Save and continue to review" button
-    Then The page title contains "Review and publish the message - MOT testing service"
+    And The page title contains "Review and publish the message - MOT testing service"
+    And The page contains "Test SN 1"
+    And The page contains "This is a test special notice"
+    And The page contains "Message successfully saved as a draft"
+
+    #Verify that the SN can be previewed by clicking the button
+    And I click the "Preview" link
+    And The page contains "Test SN 1"
+    And The page contains "This is a test special notice"
+    And The page contains "By acknowledging this message you agree to test to the standards detailed in this special notice"
+
+    #Return to review message screen, go to the homepage and return to the new inbox (saves as a draft)
+    And I click the "Back to review message" link
+    And The page title contains "Review and publish the message - MOT testing service"
     And I click the "Return to home" link
-    And I click the first "Special notices" link
```

Comment:
Can we add log out and log back in?

Reply:
Agree with this - maybe a future improvement